docs/api/event-definition.md

```diff
@@ -1,31 +1,81 @@
 ### `EventDefinition`
 
-`Object` used by Redux-Beacon to generate an event or an array of events
-for a given action.
+A `function` used by Redux Beacon to generate an event or series of events for a
+given Redux action. An event definition receives the state of the application
```

Comment:
Yup. This sentence isn't meant to specify the order that they are passed in, just that they _are_ passed in. I can take a second look if its confusing though.